app/ide-desktop/lib/content/src/remoteLog.ts

```diff
@@ -49,24 +51,26 @@ export async function remoteLog(
     message: string,
     metadata: unknown
 ): Promise<void> {
-    try {
-        const headers: HeadersInit = [
-            ['Content-Type', 'application/json'],
-            ['Authorization', `Bearer ${accessToken}`],
-        ]
-        const body = JSON.stringify({ message, metadata })
-        const response = await fetch(REMOTE_LOG_URL, { method: 'POST', headers, body })
-        if (!response.ok) {
-            const errorMessage = `Error while sending log to a remote: Status ${response.status}.`
-            try {
-                const text = await response.text()
-                throw new Error(`${errorMessage} Response: ${text}.`)
-            } catch (error) {
-                throw new Error(`${errorMessage} Failed to read response: ${String(error)}.`)
+    if (REMOTE_LOG_URL != null) {
+        try {
+            const headers: HeadersInit = [
+                ['Content-Type', 'application/json'],
+                ['Authorization', `Bearer ${accessToken}`],
+            ]
+            const body = JSON.stringify({ message, metadata })
+            const response = await fetch(REMOTE_LOG_URL, { method: 'POST', headers, body })
+            if (!response.ok) {
+                const errorMessage = `Error while sending log to a remote: Status ${response.status}.`
+                try {
+                    const text = await response.text()
+                    throw new Error(`${errorMessage} Response: ${text}.`)
+                } catch (error) {
+                    throw new Error(`${errorMessage} Failed to read response: ${String(error)}.`)
```

Comment:
I realize this `try/catch` was here before the PR but this seems like it should be a `try/catch/finally` because wouldn't the first `throw` statement get caught by the `catch` and thus it'd say `FOO Failed to read response: BAR Response: BAZ` which is misleading since we *did* read the response?